dev/com.ibm.websphere.appserver.features/visibility/public/jpaContainer-2.2/resources/l10n/com.ibm.websphere.appserver.jpaContainer-2.2.properties

```diff
@@ -14,7 +14,6 @@
 #NLS_MESSAGEFORMAT_NONE
 #
 
-description=This feature enables support for applications that use application-managed\
-and container-managed JPA written to the Java Persistence API 2.2 specification.  This \
-only includes Java Persistence API 2.2 specification interfaces, and container-managed\
-JPA integration.  This feature does not include any JPA implementation.
\ No newline at end of file
+description=This feature enables support for applications that use application-managed \
+and container-managed JPA. This only includes Java Persistence API 2.2 specification interfaces \
```

Comment:
**change**
`This only includes Java Persistence API 2.2 specification interfaces and container-managed JPA integration.`
**to**
`This support includes only Java Persistence API 2.2 specification interfaces and container-managed JPA integration.`